modules/dynamic-plug-ins-get-started.adoc

```diff
@@ -6,22 +6,14 @@
 [id="getting-started-with-dynamic-plugins_{context}"]
 = Getting started with dynamic plug-ins
 
-To get started using the dynamic plug-in, you must set up your environment to write a new OpenShift Console dynamic plug-in.
+To get started using the dynamic plug-in, you must set up your environment to write a new OpenShift Console dynamic plug-in. Dynamic plug-ins are disabled by default. In order to enable, a cluster administrator will need to enable them in the `console-operator` configuration. 
```

Comment:
Would be good to demonstrate how to enable the plugin via the console-operator
Check https://github.com/openshift/console/tree/master/dynamic-demo-plugin#enabling-the-plugin